routers/repo/download.go

```diff
@@ -22,14 +23,19 @@ func ServeData(ctx *context.Context, name string, reader io.Reader) error {
 		buf = buf[:n]
 	}
 
-	if !base.IsTextFile(buf) {
-		if !base.IsImageFile(buf) {
-			ctx.Resp.Header().Set("Content-Disposition", "attachment; filename=\""+path.Base(ctx.Repo.TreePath)+"\"")
-			ctx.Resp.Header().Set("Content-Transfer-Encoding", "binary")
-		}
-	} else if !ctx.QueryBool("render") {
+	ctx.Resp.Header().Set("Cache-Control", "public,max-age=86400")
+
+	// Google Chrome dislike commas in filenames, so let's change it to a space
+	name = strings.Replace(name, ",", " ", -1)
```

Comment:
*I* dislike spaces in filenames, how about a dash ? :)
Actually, how about a sanitizing function to generalize the issue ? Like, I'm thinking slashes might be impractical to have...